src/components/Label/__stories__/label.stories.mdx

```diff
@@ -138,5 +140,18 @@ In case of visual overload, use the secondary label in order to create hirarchy.
   </Story>
 </Canvas>
 
+### Clickable
+Default prop is set to false. In case you want to activate, change boolean to true.
```

Comment:
This description might be misleading, we might want to change it because it's not a boolean prop